packages/bot/src/scratch/help-content/flyout-help-base.jsx

```diff
@@ -69,6 +69,11 @@ class HelpBase extends React.PureComponent {
                         <Arrow2Icon />
```

Comment:
This back Icon look a bit off on the help content header. Can you add `vertical-align: middle` style for this Icon